packages/forms/src/styled/common/StyledFieldset.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'forms.fieldset';
+
+/**
+ * 1. Resets for <fieldset>.
+ */
+export const StyledFieldset = styled.fieldset.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  margin: 0; /* [1] */
+  border: 0; /* [1] */
+  padding: 0; /* [1] */
+
+  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+`;
```

Comment:
This should just be...

```js
export const StyledFieldset = styled(StyledField).attrs({
  as: 'fieldset',
  'data-garden-id': COMPONENT_ID,
  'data-garden-version': PACKAGE_VERSION
})`
  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
`;
```

It would also be consistent to include a `StyledFieldset.spec.tsx` that simply tests for a `'FIELDSET'` element rendering.